cypress/plugins/index.js

```diff
@@ -119,15 +119,20 @@ module.exports = (on, config) => {
               .createCustomToken('cypress-firestore-test-user')
               .then((token) => testApp.auth().signInWithCustomToken(token));
       const deletePromise = deleteTestData(currentTestRoot, testAdminApp);
-      const documentPath = 'disaster-metadata/2017-harvey';
-      const harveyDoc = prodApp.firestore().doc(documentPath);
-      const documentReference = testApp.firestore().doc(
-          'test/' + currentTestRoot + '/' + documentPath);
-      return Promise.all([harveyDoc.get(), signinPromise, deletePromise])
-          .then((result) => Promise.all([
-            documentReference.set(result[0].data(), {merge: true}),
-            documentReference.set({dummy: true}, {merge: true}),
-          ]))
+      const writePromises = [];
+      for (const disaster of ['2017-harvey', '2018-michael']) {
+        const documentPath = 'disaster-metadata/' + disaster;
+        const prodDisasterDoc = prodApp.firestore().doc(documentPath);
+        const testDisasterDocReference = testApp.firestore().doc(
+            'test/' + currentTestRoot + '/' + documentPath);
+        writePromises.push(
+            Promise.all([prodDisasterDoc.get(), signinPromise, deletePromise])
```

Comment:
Is it weird to wait on these twice in a sequential loop?

Reply:
They're not actually being waited on in the loop, they're just being put into a Promise.all. We then wait on them in parallel in the Promise.all below. So the graph of Promise waiting looks like:
![image](https://user-images.githubusercontent.com/10134896/69366810-0a2a5c80-0c65-11ea-9760-b516ee7f93d8.png)

Note that we construct that entire graph before we do any waiting, so there's no sequential waiting.